### ISSUE TITLE
systemd-resolv and sound hangs after running xwd -root

### ISSUE THREAD

**Issue by tecan** [mate-desktop/mate-terminal#208]
#### Expected behaviour
terminal full of chars

#### Actual behaviour
locks up system and eats up cpu after it unfreezes

#### Steps to reproduce the behaviour
open up any terminal in mate desktop and run xwd -root

**Comment by egmontkob** [comment]
(Not mate-terminal but VTE developer here.)

At this moment I can imagine this the X server getting overloaded or locked, or mate-terminal starting to eat memory on some byte sequence. Let's first see which of these two happens.

What do you mean by "any terminal"? Is this behavior reproducible in let's say xterm or urxvt?

What happens if you redirect the output to a file, like `xwd -root > /tmp/foobar`?

What if you later cat that file: `cat /tmp/foobar`?

What VTE version do you have? E.g. what four-digit number appears as if you typed it, if you run `echo -ne '\e[>c'`?

**Comment by tecan** [comment]
1;4804;0c was what i got for vte , yes it pipes to file nicely then if i cat the file it causes it, so i can now rule out the sound system. have only tried with vte and mate-terminal sofar.

**Comment by egmontkob** [comment]
Thanks; that's VTE version 0.48.4. I'm on 0.51-ish but there weren't any changes I can think of right now that could be relevant here.

What occurred to me right now is that gnome-terminal or VTE received a couple of bugreports about ringing the bell too much DoS'ing or crashing pulseaudio. Could you please print the file, with the BEL characters omitted: `tr -d '\a' < /tmp/foobar`. Does it work correctly?

**Comment by tecan** [comment]
yes that worked fine, another fellow mentioned that systemd-resolve might be the actual culprit due to could possibly be exploited for privilege escalation or hiding process from users.

**Comment by tecan** [comment]
also when i ran the tr method there were some leftover chars in the new bash input line. 62;c62;c

**Comment by egmontkob** [comment]
Okay, so it's flooding the system bell that causes the trouble. Time for us to do something about it. Thanks for the help in the investigation!

Those 62;c stuff are irrelevant and harmless, although sure annoying as hell. Some escape sequences respond by placing something in the input queue as if it was typed from the keyboard. Don't worry, you can't inject "rm -rf" this way :)

**Comment by egmontkob** [comment]
Filed [Bell rate limit](https://bugzilla.gnome.org/show_bug.cgi?id=790815) VTE bugreport.

**Comment by flexiondotorg** [comment]
Thanks @egmontkob
